cypress/support/page_objects/homePage.js

```diff
@@ -14,20 +14,27 @@ class ChatBot{
     openChatBot(){
         cy.frameLoaded('#chat-widget', { url: '/secure.livechatinc.com' })
         cy.wait(1000)
+
     }
 
     startChatBot(first_name, mobile_phone, email) {
         cy.enter('#chat-widget').then(getBody => {
             getBody().find('[aria-label="Open LiveChat chat widget"]').should('be.visible').click()
             getBody().find('#name').type(first_name)
-            getBody().contains('WhatsApp Number').should('be.visible').type(mobile_phone)
+            getBody().find('.Linkify').contains('WhatsApp Number').should('be.visible').type(mobile_phone)
```

Comment:
You should find an unique selector for elements (this might works the first time but it fails when run it again)